subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencySubstitutionRulesIntegrationTest.groovy

```diff
@@ -32,6 +32,7 @@ class DependencySubstitutionRulesIntegrationTest extends AbstractIntegrationSpec
         resolve.addDefaultVariantDerivationStrategy()
     }
 
+    @ToBeFixedForInstantExecution(because = "Task with Configuration field in ResolveTestFixture")
```

Comment:
🔴 I'd change the message to include the name of the offending task and save the reader some time, maybe something like `because = "Configuration field of GenerateGraphTask used by ResolveTestFixture"`.

Reply:
Not applicable anymore with the recent changes, resolving